tests/integration/trace/test_zipkin_trace.py

```diff
@@ -127,7 +124,7 @@ def test_zipkin_trace(trace_server):
 
 
 @pytest.mark.gen_test
-def test_tcollector_submit(monkeypatch, trace_server):
+def test_tcollector_submit(trace_server):
```

Comment:
No. This unit test is test zipkin trace's submit method. It is not an end to end test. We have 'test_zipkin_trace' test doing end to end test for zipkin. In this test, we register zipkintracehook.